$OpenBSD: patch-bin_passenger-config,v 1.1 2011/12/13 18:38:17 jeremy Exp $

Make sure that --root uses the location of the symlink, so it
doesn't change from version to version.

--- bin/passenger-config.orig	Thu Dec  1 07:30:31 2011
+++ bin/passenger-config	Thu Dec  1 07:30:53 2011
@@ -37,7 +37,7 @@ end
 
 case ARGV[0]
 when "--root"
-	puts PhusionPassenger::SOURCE_ROOT
+	puts '${PASSENGER_ROOT}'
 when "--version"
 	puts PhusionPassenger::VERSION_STRING
 else
